lib/puppet/ssl/certificate_authority.rb

```diff
@@ -441,11 +441,11 @@ def x509_store(options = {})
   # Certificate Revocation List and flags
```

Comment:
It looks like `x509_store`, defined on line 430 calls `create_x509_store`. It's a deprecated method, but it'll still error out if anyone tries to use it to create a store after this change. Should we update it to pass in a purpose as well? Or just leave it broken?

Reply:
nevermind, this is what I get for looking at commits one at a time. I see you added a default for purpose in the next commit, so this is no longer a concern.

Reply:
no that's a valid concern. I should have added the default value in the previous commit.

Reply:
well, I guess we're leaving it as is :)